fabtests/component/sock_test.c

```diff
@@ -0,0 +1,253 @@
+#include <stdio.h>
+#include <string.h>  
+#include <stdlib.h>
+#include <errno.h>
+#include <unistd.h>   
+#include <arpa/inet.h>    
+#include <sys/types.h>
+#include <sys/socket.h>
+#include <netinet/in.h>
+#include <sys/time.h>
+#include <netdb.h>
+#include <time.h>
+#include <poll.h>
+#include <sys/epoll.h>
+
+#define PORT 8888

```

Comment:
Update test to use some common fabtests structures.  E.g. use default_port value and ft_opts.  The latter allows reuse of address options (see below).